src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSTaxonomyDatabase.java

```diff
@@ -1,16 +1,66 @@
 package org.broadinstitute.hellbender.tools.spark.pathseq;
 
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+
+import java.util.HashMap;
 import java.util.Map;
 
 /**
  * Helper class for holding taxonomy data used by ClassifyReads
  */
+@DefaultSerializer(PSTaxonomyDatabase.Serializer.class)
 public class PSTaxonomyDatabase {
     public final PSTree tree;
-    public final Map<String, String> contigToTaxIDMap; //Reference contig name to taxonomic ID
+    public final Map<String, String> accessionToTaxId; //Reference contig name to taxonomic ID
 
     public PSTaxonomyDatabase(final PSTree tree, final Map<String, String> map) {
         this.tree = tree;
-        this.contigToTaxIDMap = map;
+        this.accessionToTaxId = map;
+    }
+
+    private PSTaxonomyDatabase(final Kryo kryo, final Input input) {
```

Comment:
Since this is a pretty complicated serialization/deserialization, it'd be nice to write a unit test that exercises a round trip of this object with a small tree attached.